src/lib/import-export/import/importers/importer.ts

```diff
@@ -71,9 +71,42 @@ export abstract class BaseImporter extends EventEmitter implements Importer {
 			}
 		}
 
+		this.replaceSiteUrl( siteId );
```

Comment:
Since `replaceSiteUrl` is an asynchronous function, we should wait for the execution.

```suggestion
		await this.replaceSiteUrl( siteId );
```